--- ./lib/RT/Interface/Email.pm	2025-05-28 08:40:28.000000000 -0400
+++ ../Email.pm	2025-08-05 11:50:08.139485877 -0400
@@ -975,14 +975,29 @@
     } elsif ( $mail_command eq 'mbox' ) {
         my $now = RT::Date->new(RT->SystemUser);
         $now->SetToNow;
-
+    
         state $logfile;
         unless ($logfile) {
+            my $mbox_dir = "$RT::VarPath/mbox";
+            unless (-d $mbox_dir) {
+                require File::Path;
+                File::Path::make_path($mbox_dir, {
+                    mode => 0755,
+                    error => \my $err
+                });
+                if (@$err) {
+                    $RT::Logger->crit("Can't create mbox directory $mbox_dir: " . 
+                        join(', ', map { $_->{$_->{file}}} @$err));
+                    return 0;
+                }
+            }
+            
             my $when = $now->ISO( Timezone => "server" );
             $when =~ s/\s+/-/g;
-            $logfile = "$RT::VarPath/$when.mbox";
+            $logfile = "$mbox_dir/$when.mbox";
             $RT::Logger->info("Storing outgoing emails in $logfile");
         }
+    
         my $fh;
         unless (open($fh, ">>", $logfile)) {
             $RT::Logger->crit( "Can't open mbox file $logfile: $!" );
